libcpuid/libcpuid.h

```diff
@@ -831,7 +832,7 @@ struct msr_driver_t* cpu_msr_driver_open(void);
  *          The error message can be obtained by calling \ref cpuid_error.
  *          @see cpu_error_t
  */
-struct msr_driver_t* cpu_msr_driver_open_core(int core_num);
+struct msr_driver_t* cpu_msr_driver_open_core(uint8_t core_num);
```

Comment:
This value need to be unsigned (a core number is always positive). It avoid `cpu_msr_driver_open_core()` to check if value is positive.
8-bit value because a range of 0 to 255 is correct.